client/new-stack-editor/lib/index.coffee

```diff
@@ -47,3 +62,15 @@ module.exports = class StackEditorAppController extends AppController
     console.trace()
     log '::reloadEditor', templateId, skipDataUpdate
 
+
+  fetchStackTemplate: (templateId, callback) ->
+
+    if template = @templates[templateId]
+      return callback null, template
+
+    cc = kd.singletons.computeController
+    cc.fetchStackTemplate templateId, (err, template) =>
+      return callback err  if err
+      return callback Errors.NotExists  unless template
```

Comment:
Minor thing, this line will not run, fetchStackTemplate is already checking the template existency. [here](https://github.com/koding/koding/blob/master/client/app/lib/providers/computecontroller.coffee#L1216)